t/t1001-mf-priority.t

```diff
@@ -0,0 +1,127 @@
+#!/bin/bash
+
+test_description='Test multi-factor priority plugin'
+
+. `dirname $0`/sharness.sh
+MULTI_FACTOR_PRIORITY=${FLUX_BUILD_DIR}/src/plugins/.libs/mf_priority.so
+# BULK_UPDATE=${FLUX_BUILD_DIR}/t/scripts/send_fake_payloads.py
+
+export TEST_UNDER_FLUX_NO_JOB_EXEC=y
+export TEST_UNDER_FLUX_SCHED_SIMPLE_MODE="limited=1"
+test_under_flux 1 job
+
+flux setattr log-stderr-level 1
+
+test_expect_success 'load multi-factor priority plugin' '
+		flux jobtap load -r .priority-default ${MULTI_FACTOR_PRIORITY}
```

Comment:
Hm, now all the indents seem to be 2 tabs? 
```suggestion
	flux jobtap load -r .priority-default ${MULTI_FACTOR_PRIORITY}
```

Reply:
Oops, I had always had the indents at 2 tabs. These tests should now all be indented at one tab (4 spaces).

Reply:
Yeah, it is confusing. I have my editor set to 8 space tabs (at least for the sharness tests), so for me the lines were indented 16 spaces! A single tab was probably plenty. :slightly_smiling_face: